test-scripts/tomcat-deployment/ansible/hosts

```diff
@@ -0,0 +1,5 @@
+[tomcat-servers:vars]
+ansible_ssh_private_key_file=./key.pem
+
+[tomcat-servers]
+54.224.223.61
```

Comment:
It seems this file is re-generated by deploy.sh. So, shall we add a comment asking users to not modify this manually?